routes/_components/profile/AccountProfileDetails.html

```diff
@@ -8,20 +8,24 @@
     </span>
   </div>
   <div class="account-profile-details-item">
-    <span class="account-profile-details-item-title">
-      Follows
-    </span>
-    <span class="account-profile-details-item-datum">
-      {{numFollowingDisplay}}
-    </span>
```

Comment:
we might want to put the class on the `<a>` instead of the `<span>`, and probably remove the `<span>` altogether since it'll no longer be necessary

Reply:
hmm there are two `span`s and two classes though? one for the title and one for the value. or did you mean the wrapping `div` and its class?

Reply:
d'oh, my bad. you're right.